dspace-server-webapp/src/main/java/org/dspace/app/rest/model/RestAddressableModel.java

```diff
@@ -44,4 +44,9 @@ public Projection getProjection() {
     public void setProjection(Projection projection) {
         this.projection = projection;
     }
+
+    @JsonIgnore
+    public String getUniqueType() {
```

Comment:
I want to talk more about this method in tomorrow's meeting. But, this method should have JavaDocs to describe it, and especially how it differs from the existing [`RestModel.getType()` method](https://github.com/DSpace/DSpace/blob/master/dspace-server-webapp/src/main/java/org/dspace/app/rest/model/RestModel.java#L35) (and when to use one over the other).

Reply:
as agreed in the dev meeting I have created a ticket to cleanup that later
https://jira.lyrasis.org/browse/DS-4460